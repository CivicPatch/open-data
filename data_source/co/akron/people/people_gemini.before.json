[
  {
    "name": "Brandon Hill",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Jared Jefferson",
    "roles": [
      "Trustee",
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Crystann Benson",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Braden Brent",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Ariella Gonzales-Vondy",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Jennifer Hansen",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "David Kembel",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  },
  {
    "name": "Gillian Laycock",
    "roles": [
      "Town Manager"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "970-554-0832",
    "email": "glaycock@townofakron.com",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://townofakron.colorado.gov/home/employment",
      "https://townofakron.colorado.gov/government/agenda-minutes-current"
    ],
    "updated_at": "2025-07-09T00:16:44Z"
  }
]