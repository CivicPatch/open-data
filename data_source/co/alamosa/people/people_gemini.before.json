[
  {
    "name": "Jackie Vigil",
    "roles": [
      "council member",
      "Councilor"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": null,
    "phone_number": "(719) 992-8008",
    "email": null,
    "website": "https://cityofalamosa.org/mayorprotem-ward3",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/mayorprotem-ward3",
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/mayor"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Ty Coleman",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(719) 313-7241",
    "email": null,
    "website": "https://cityofalamosa.org/mayor",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/mayor"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Michael Carson",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "at Large"
    ],
    "image": null,
    "phone_number": "(719) 992-4839",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-daniel",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/councilor-at-large-daniel"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Jan Vigil",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "at Large"
    ],
    "image": null,
    "phone_number": "(970) 946-5611",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-vigil",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/councilor-at-large-vigil"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Liz Hensley",
    "roles": [
      "Councilor",
      "City Councilor"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": null,
    "phone_number": "(719) 580-0201",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-1",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-1"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Dawn Krebs",
    "roles": [
      "Councilor",
      "City Councilor"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": null,
    "phone_number": "(719) 580-2194",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-2",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-2"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Jamie Dominguez",
    "roles": [
      "Councilor",
      "City Councilor"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": null,
    "phone_number": "(719) 937-3039",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-4",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-2",
      "https://cityofalamosa.org/city-councilor-ward-4"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  }
]