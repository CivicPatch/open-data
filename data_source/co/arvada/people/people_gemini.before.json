[
  {
    "name": "Brad Rupert",
    "roles": [
      "council member",
      "Councilmember"
    ],
    "divisions": [
      "at-large",
      "At-Large"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": "2024-12-16",
    "end_date": "2025-11",
    "sources": [
      "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Lauren Simpson",
    "roles": [
      "mayor",
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "John Marriott",
    "roles": [
      "Councilmember"
    ],
    "divisions": [
      "District 3"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Bob Fifer",
    "roles": [
      "Councilmember"
    ],
    "divisions": [
      "District 4"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Randy Moorman",
    "roles": [
      "Mayor Pro Tem"
    ],
    "divisions": [
      "District 1"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Sharon Davis",
    "roles": [
      "Councilmember"
    ],
    "divisions": [
      "At-Large"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Shawna Ambrose",
    "roles": [
      "Councilmember"
    ],
    "divisions": [
      "District 2"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  }
]