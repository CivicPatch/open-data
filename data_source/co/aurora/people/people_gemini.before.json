[
  {
    "name": "Mike Coffman",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "mcoffman@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448210",
    "start_date": "2023",
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/mike_coffman",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448210"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Crystal Murillo",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward I"
    ],
    "image": null,
    "phone_number": "303.739.7502",
    "email": "cmurillo@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448231",
    "start_date": null,
    "end_date": "2025-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/crystal_murillo",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448231"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Steve Sundberg",
    "roles": [
      "Council Member",
      "Mayor Pro Tem"
    ],
    "divisions": [
      "Ward II"
    ],
    "image": null,
    "phone_number": "303.525.6659",
    "email": "Steve.Sundberg@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174017",
    "start_date": null,
    "end_date": "2025-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/steve_sundberg",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174017"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Ruben Medina",
    "roles": [
      "Council Member",
      "council member"
    ],
    "divisions": [
      "Ward III"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "rlmedina@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174140",
    "start_date": null,
    "end_date": "2025-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/ruben_medina",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174140"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Stephanie Hancock",
    "roles": [
      "Council Member",
      "council member"
    ],
    "divisions": [
      "Ward IV"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "shancock@auroragov.org",
    "website": "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/stephanie_hancock",
    "start_date": null,
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/stephanie_hancock"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Angela Lawson",
    "roles": [
      "Council Member",
      "Chair",
      "Vice Chair",
      "Member"
    ],
    "divisions": [
      "Ward V"
    ],
    "image": null,
    "phone_number": "303.739.7518",
    "email": "alawson@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448434",
    "start_date": null,
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/angela_lawson",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448434"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Fran√ßoise Bergan",
    "roles": [
      "Council Member",
      "council member"
    ],
    "divisions": [
      "Ward VI"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "fbergan@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448317",
    "start_date": null,
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/francoise_bergan",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448317"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Alison Coombs",
    "roles": [
      "Council Member",
      "council member",
      "Chair",
      "Vice Chair"
    ],
    "divisions": [
      "At Large"
    ],
    "image": null,
    "phone_number": "720.505.3332",
    "email": "acoombs@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448307",
    "start_date": null,
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/alison_coombs",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448307"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Curtis Gardner",
    "roles": [
      "Council Member",
      "Chair",
      "Vice Chair"
    ],
    "divisions": [
      "At Large"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "cgardner@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448377",
    "start_date": null,
    "end_date": "2027-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/curtis_gardner",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448377"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Danielle Jurinsky",
    "roles": [
      "Council Member",
      "council member",
      "Chair",
      "Vice Chair"
    ],
    "divisions": [
      "At Large"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "djurinsk@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174231",
    "start_date": null,
    "end_date": "2025-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/danielle_jurinsky",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174231"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  },
  {
    "name": "Amsalu Kassaw",
    "roles": [
      "Council Member",
      "Chair",
      "Vice Chair"
    ],
    "divisions": [
      "At Large"
    ],
    "image": null,
    "phone_number": "303.739.7015",
    "email": "akassaw@auroragov.org",
    "website": "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=20651707",
    "start_date": null,
    "end_date": "2025-11",
    "sources": [
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members",
      "https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/amsalu_kassaw",
      "https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=20651707"
    ],
    "updated_at": "2025-06-27T19:42:57Z"
  }
]