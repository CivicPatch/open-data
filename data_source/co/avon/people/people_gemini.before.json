[
  {
    "name": "Tamra Underwood",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "tunderwood@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=163",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Rich Carroll",
    "roles": [
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "rcarroll@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=7",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=7"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Chico Thuon",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "cthuon@avon.org",
    "website": null,
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Lindsay Hardy",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "lhardy@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=204",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=204"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Ruth Stanley",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "rstanley@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=339",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=339"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Gary Brooks",
    "roles": [
      "Town Council Member",
      "Town Councilmember"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "gbrooks@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=373",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=373"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Kevin Hyatt",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "khyatt@avon.org",
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  }
]