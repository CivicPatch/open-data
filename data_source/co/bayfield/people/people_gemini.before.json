[
  {
    "name": "Tom Au",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Alexis Hartz",
    "roles": [
      "Mayor Pro-Tem"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Brenna Morlan",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Matthew Nyberg",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Laura MacLaurin",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Kat Katsos",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2024",
    "end_date": "2026",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  },
  {
    "name": "Lori Zazzaro",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": "2024",
    "end_date": "2026",
    "sources": [
      "https://www.bayfieldgov.org/board-trustees"
    ],
    "updated_at": "2025-06-30T22:34:38Z"
  }
]