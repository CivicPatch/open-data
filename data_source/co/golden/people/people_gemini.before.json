[
  {
    "name": "Bill Fisher",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "Ward Four",
      "Ward 4"
    ],
    "image": null,
    "phone_number": null,
    "email": "BFisher@cityofgolden.net",
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net",
      "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Laura Weinberg",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Lisa Vitry",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "District One"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Patty Evans",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "District Two"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Robert Reed",
    "roles": [
      "Councilor",
      "Mayor Pro-Tem"
    ],
    "divisions": [
      "Ward One"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Paul Haseman",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "Ward Two"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Don Cameron",
    "roles": [
      "Councilor"
    ],
    "divisions": [
      "Ward Three"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  }
]